### ISSUE TITLE
Importing Nmap output does not create nodes 

### ISSUE THREAD

**Issue by aim4r** [dradis/dradis-ce#26]
Using Dradid CE 3.1.0-rc2 on Kali Linux (all packages updated to the latest).
Importing Nmap scan results does not create any nodes and/or issues.

**Comment by ooctav** [comment]
Hi, I am also experiencing the same problem, with a nmap XML file (generated with `nmap -sS a.b.c.d/24 -oX log.xml`). 

The `log.xml` file is uploaded (approx. 1.3MB) - I do get the following message from the Upload Manager page:
`
[13:25:03] Enqueueing job to start in the background. Job id is 5391e6ab240b11b76b3bbec3cc8bc1f5
`

Unfortunately, I cannot get any nodes / issues to be created. Also worth noting that the log files available, resque.log and development.log show no relevant information on any background jobs.

To sum it up:
* nmap, Windows binaries, version 7.01
* nmap CLI: `nmap -sS a.b.c.d/24 -oX log.xml`
* ruby: `ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-linux]`
* dradis: on commit cb51a3ad4b638f15f1710a863d768becaddefd43


Thanks!

**Comment by rachkor** [comment]
@aim4r @ooctav Can you check out this thread in the community discussion forum? http://discuss.dradisframework.org/t/issues-with-uploads/328/2
If all looks good with the background worker, try uploading the file from the command line and let us know if you get any error messages.

**Comment by ooctav** [comment]
@rachkor It worked: starting the resque worker (manually), via `bundle exec rake resque:work` has made the background task work. I missed this when reading the _Installing via git_ section in the documentation.

Thank you!

**Comment by etdsoft** [comment]
Closing this now.
